docs/deployment.md

```diff
@@ -0,0 +1,90 @@
+# Deployments
+
+## Next.js app
+
+The Next.js app can be deployed as a static HTML export or as a Node.js server, depending on your needs. The [Next.js deployment documentation](https://nextjs.org/docs/deployment) provides more information on how to deploy your application.
+
+## Storybook
+
+Storybook can be deployed as a static HTML export. The [Storybook deployment documentation](https://storybook.js.org/docs/react/sharing/publish-storybook) provides more information on deployment options.
+
+### GitHub Pages
+
+[GitHub Pages](https://pages.github.com/) may be a good option for hosting Storybook if you only need one environment to represent the latest state of your main branch.
+
+**Steps to setup GitHub Pages**
+
+1. Add a workflow to automatically deploy Storybook: `.github/workflows/deploy-storybook.yml` (see example below)
+1. Enable the "GitHub Actions" source in your rep settings (`Settings > Pages > Source`)
+1. Trigger the workflow from the Actions tab or by pushing a commit to your main branch
+
+---
+
+<details>
+  <summary>View GitHub workflow</summary>
+
+```yaml
```

Comment:
Is there anything in this workflow that wouldn't work out of the box? what do you think about putting a file in `.github/workflows/` and then commenting out the push trigger with instructions to uncomment? i think instructions to uncomment something is easier to do than instructions to create a new file. I don't feel strongly about it either way in this case.

Reply:
Sounds good. Added a workflow that's disabled by default.